src/caip.ts

```diff
@@ -16,6 +16,39 @@ export const CaipNamespaces = {
   Eip155: 'eip155' as CaipNamespace,
 } as const;
 
+/** Function signature for CAIP reference normalizers. */
+export type CaipReferenceNormalizer = (reference: string) => string;
+
+/**
+ * Normalize a eip155 CAIP reference.
+ *
+ * @param reference - The eip155 CAIP reference to be normalized.
```

Comment:
```suggestion
 * @param reference - The EIP-155 CAIP reference to be normalized.
```

Reply:
After discussing together, I removed this part!